dev_requirements.txt

```diff
@@ -21,6 +21,8 @@ pydata-sphinx-theme~=0.8.1
 jupytext==1.11.1
 sphinx-gallery
 nbsphinx
+openfermion
+openfermionpyscf
```

Comment:
Are these required by PennyLane?  If so, please use:
```
openfermion==1.3.0; sys_platform != 'win32'
openfermionpyscf==0.5; sys_platform != 'win32'
```
as in the latest version of the Mitiq master branch.